client/src/pages/Interview.js

```diff
@@ -98,6 +102,30 @@ const Interview = (props) => {
 		props.history.push('/dashboard');
 	};
 
+	const compileCode = async () => {
+		let extension = 'js';
```

Comment:
Just an idea, you don't have to do it! Maybe you could add a `loading` state and pass it down to the console. We can set it to true here, so that while we wait for the results of the compilation, we can say `loading` or something in the console.

Reply:
That's a nice idea, to my surprise it was very easy to implement.